src/pages/fr/guides/rss.md

```diff
@@ -0,0 +1,161 @@
+
+> **Note :** La variable d'environnement `SITE` n'existe que dans la dernière version bêta de Astro 1.0. Mettez à jour Astro (`astro@latest`), ou écrivez votre `site` manuellement si cela n'est pas possible (voir les exemples ci-dessous).
+
+Maintenant que tout est prêt, générons notre premier flux RSS ! Créez un fichier `rss.xml.js` dans votre répertoire `src/pages/`. `rss.xml` sera l'URL de sortie, donc n'oubliez pas de le renommer si vous préférez.
+
+Ensuite, importez l'Helper `rss` du Package `@astrojs/rss` et appelez-le avec les paramètres suivants :
+
+```js
+// Exemple: src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+export const get = () => rss({
+    // la valeur `<title>` de votre XML de sortie
+    title: 'Buzz’s Blog',
+    // la valeur `<description>` de votre XML de sortie
+    description: 'A humble Astronaut’s guide to the stars',
+    // l'URL de base pour les liens RSS <item>
+    // SITE sera utilisé depuis votre valeur "site" du fichier astro.config de votre projet.
+    site: import.meta.env.SITE,
+    // la liste des `<item>`s dans votre XML de sortie
+    // exemple simple : générer des items pour chaque fichier ".md" dans "/src/pages"
+    // voir la section "Génération des items" pour les frontmatter requis et les cas avancés
+    items: import.meta.glob('./**/*.md'),
+    // (optionnel) injecter des valeurs XML personnalisés
+    customData: `<language>en-us</language>`,
+  });
+```
+
+### Génération des `items`
+
+Les `items` acceptent soit :
+
+1. [Un résultat de `import.meta.glob(...)`](#1-avec-un-résultat-importmetaglob) **(utilisez cette option pour les fichiers `.md` à l'interieur du répertoire `src/pages/`)**
+2. [Une liste d'objets RSS](#2-avec-une-liste-dobjets-rss), chacun possédant un champ `link`, `title`, `pubDate`, et optionnellement un champ `description` et/ou `customData`.
+
+#### 1. Avec un résultat `import.meta.glob`
+
+Nous recommandons cette option comme une option plus rapide pour les fichiers `.md` dans le répertoire `src/pages/`. Chaque article devrait avoir un champ `title`, `pubDate`, et un champ optionnel `description` et/ou `customData` dans son Frontmatter. Si cela n'est pas possible, ou si vous préférez générer ce frontmatter en code, [voir l'option 2](#2-avec-une-liste-dobjets-rss).
+
+Disons que vos articles sont stockés dans le répertoire `src/pages/blog/`. Vous pouvez générer un flux RSS comme suit :
+
+```js
+// Exemple: src/pages/rss.xml.js
+import rss from '@astrojs/rss';
+export const get = () => rss({
+  title: 'Buzz’s Blog',
+  description: 'A humble Astronaut’s guide to the stars',
+  site: import.meta.env.SITE,
+  items: import.meta.glob('./blog/**/*.md'),
+});
+```
+
+Jetez un oeil à la [documentation de l'import glob de Vite](https://vitejs.dev/guide/features.html#glob-import) pour plus d'informations sur sa syntaxe.
```

Comment:
Jetez un oeil à la [documentation de l'import glob de Vite](https://vitejs.dev/guide/features.html#glob-import) pour plus d'information sur cette syntaxe d'import.

Reply:
Je pense avoir écrit le commentaire un peu trop souvent, j'en ai fait un post'it dans un coin 🤣